src/main/java/com/icthh/xm/ms/configuration/web/rest/ConfigurationAdminResource.java

```diff
@@ -84,7 +84,7 @@
         return ResponseEntity.ok().build();
     }
 
-    @GetMapping(value = CONFIG + "/**")
+    @GetMapping(value = CONFIG + "/**", produces = {APPLICATION_JSON_VALUE, APPLICATION_JSON_UTF8_VALUE, TEXT_PLAIN_VALUE})
```

Comment:
better to remove produces
how about xml etc...?

Reply:
@vkirichenko please comment as you helped me to investigate this issue.

Reply:
not need anymore, removed